### ISSUE TITLE
View Default Values on Logix Node Inputs

### ISSUE THREAD

**Issue by H3BO3** [Frooxius/NeosPublic#1047]
It would be neat to be able to see the default values of node inputs, perhaps on the tooltip's text while hovering over them. While most inputs default to False or 0, there's quite a few exceptions, and being able to see when those are present would mitigate unexpected outcomes.

**Comment by Enverex** [comment]
A good and confusing example of this is the Tween node. If you pull out the input for the tweening motion type, the selector will default to Linear, but that's not actually the default. So in cases like this, there's actually no-way to know what the default is without testing and the fact the selector defaults to a value other than the actual default is misleading.

**Comment by Frooxius** [comment]
This is going to be part of general documentation for the nodes. The default values can be context dependent and aren't necessarily known until you actually evaluate the node. E.g. when unplugged, certain nodes will default the value to whatever the current value of their target is, so they leave it unchanged. 

However doing so will need the nodes to be manually annotated and the inputs described.

**Comment by shiftyscales** [comment]
Hmm- a while back I had seen an old post you had made in 2018 about having a value 'peek' on the Logix tip.
https://discordapp.com/channels/402159838827905024/402159839251398666/448192344873762828
> One thing I want to do as well is value "peek". If you point at the wire with your LogiX tool, it'll show the current value at that wire without having to add display node.
@Frooxius#4483

Do you think that such a system could be feasible, @Frooxius? That could potentially take care of this use-case if hovering at the inputs/outputs displayed their values?

**Comment by H3BO3** [comment]
That's basically what I want. Show the value that's currently present at the input/output, with the understanding that it may be different from the "default" value depending on if a ribbon is plugged in or not.

**Comment by Frooxius** [comment]
@shiftyscales @H3BO3 No, like I said, the actual default value is context dependent, so you can't really "peek" it in every case. This would require some kind of debugger for LogiX to capture in particular context.

In other words, with some nodes the default value changes depending on the context and other values being evaluated. Typically it's done to preserve whatever property already is there if the input is unplugged. This means there's not a single default value to show you, but it can only be described like "Default value preserves the xyz state of the input Slot" for example.
